warehouse/transformer/internal/utils/utils_test.go

```diff
@@ -132,10 +132,10 @@ func TestIsBlank(t *testing.T) {
 		{"FloatNonZero", 123.45, false},                             // non-zero float
 		{"BoolFalse", false, false},                                 // boolean false
 		{"BoolTrue", true, false},                                   // boolean true
-		{"EmptySlice", []int{}, false},                              // empty slice
-		{"NonEmptySlice", []int{1, 2, 3}, false},                    // non-empty slice
-		{"EmptyMap", map[string]int{}, false},                       // empty map
-		{"NonEmptyMap", map[string]int{"key": 1}, false},            // non-empty map
+		{"EmptySlice", []any{}, true},                               // empty slice
```

Comment:
Why are we changing the type from `int` to `any`? Will the tests pass with `int` type ?

Reply:
Those were invalid test cases, We don't get the typed map or slice. It will be of any type.